autotest-tw-solver.py

```diff
@@ -8,8 +8,9 @@
 exhibited bugs.
 
 Optional argument:
-    --full   run test on all graphs with min-degree 3 and at most 8 vertices
-
+    ++full   run test on all graphs with min-degree 3 and at most 8 vertices
+    ++       any number of arguments that get passed on to the treewidth solver
```

Comment:
Is the interface now like this? `./autotest-tw-solver.py mysolver ++--twsolveroption1`
Maybe a nicer interface would be: `./autotest-tw-solver.py [--full] <solver> [arguments for solver]` so that solver arguments can be given as-is, without having to prepend `++`.